liquibase-couchbase/src/main/java/liquibase/ext/couchbase/changelog/NoSqlHistoryService.java

```diff
@@ -48,10 +48,6 @@ public abstract class NoSqlHistoryService extends AbstractChangeLogHistoryServic
     private boolean initialized;
     private final Logger log = Scope.getCurrentScope().getLog(getClass());
 
-    public int getPriority() {
```

Comment:
won't it affect anythin?

Reply:
No. It's duplicate. CouchbaseHistoryService already override this method